app/main/views/g_cloud.py

```diff
@@ -708,6 +757,14 @@ def why_did_you_not_award_the_contract(framework_family, project_id):
     form = WhyDidYouNotAwardForm()
 
     if form.validate_on_submit():
+        if form.why_did_you_not_award_the_contract.data == 'work_cancelled':
+            data_api_client.create_direct_award_project_outcome_cancelled(
+                project_id=project_id,
+                user_email=current_user.email_address)
+        elif form.why_did_you_not_award_the_contract.data == 'no_suitable_services':
+            data_api_client.create_direct_award_project_outcome_none_suitable(
+                project_id=project_id,
+                user_email=current_user.email_address)
         flash('You’ve updated ‘' + project['name'] + '’')
```

Comment:
Can we use format strings for these as it will make internationalization slightly easier if we ever do that

Reply:
Will do